horizons/ai/aiplayer/goal/feederchaingoal.py

```diff
@@ -164,7 +164,7 @@ def get_personality_name(self):
 
 class FeederHygieneGoal(FeederChainGoal):
 	def __init__(self, settlement_manager):
-		super().__init__(settlement_manager, RES.HYGIENE, 'hygiene producer')
+		super(FeederHygieneGoal, self).__init__(settlement_manager, RES.HYGIENE, 'hygiene producer')
```

Comment:
Is this necessary at all? Per Python 3.x documentation, naming the child object and the binding is no longer needed:
```language="python"
class C(B):
    def method(self, arg):
        super().method(arg)    # This does the same thing as:
                               # super(C, self).method(arg)
```